composer/cli/launcher.py

```diff
@@ -164,15 +168,20 @@ def monitor_processes(processes: Set[subprocess.Popen]):
                         output=output,
                         stderr=stderr,
                     )
-                    error_msg = [
-                        "Error in subprocess",
-                        "----------Subprocess STDOUT----------",
-                        exc.output,
-                        "----------Subprocess STDERR----------",
-                        exc.stderr,
-                    ]
+                    error_msg = [f"Process {process.pid} excited with code {process.returncode}"]
+                    if output is not None:
```

Comment:
I have a preference for making it very explicit when stdout and stderr are empty, even if that looks like
```
----------Begin subprocess STDOUT----------
----------End subprocess STDOUT----------
```

Otherwise, sometimes I'm not sure as a user whether the program isn't giving me stdout, or if it's just empty.

Reply:
It will be None if it's rank 0, not if it's empty. It should still print if it's empty because `None is not ""`